src/pint/plot_utils.py

```diff
@@ -176,3 +177,127 @@ def phaseogram_binned(
         plt.close()
     else:
         plt.show()
+
+
+def plot_priors(
+    model,
+    chains,
+    fitvals,
+    fiterrs,
+    maxpost_fitvals,
+    burnin=100,
+    bins=100,
+    scale=False,
+    file=False,
+):
+
+    """
+    Show binned samples, prior probability distribution and an initial gaussian
+    probability distribution plotted with 2 sigma, maximum posterior and original
+    fit values marked.
+    
+    Parameters
+    ----------
+    model
+        Input timing model 
+    chains: dict    
+        Post MCMC integration chains that are input into a histogram and compared with the priors. Contain the fitkeys and the samples
+    fitvals
+        Initial Parameter values
+    fiterrs
+        Initial Parameter uncertainties
+    maxpost_fitvals
```

Comment:
Can these be optional? If you don't have them, can the code still make sensible plots? That would make the function a little more convenient. Actually if all the fit values and errors were optional, drawn on the plot only if provided, that would make this function more convenient - just drop the chains in if they're all you have, or include more information for a more useful plot.

Reply:
The fitvals and fiterrs are used to calculate an initial gaussian probability distribution. I can make it so that it plots just the chain values and the priors and add in optionals.